qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -45,6 +50,10 @@
 
 parsed_user_agent = None
 
+_qute_scheme_handler = cast(webenginequtescheme.QuteSchemeHandler, None)
+_req_interceptor = None
```

Comment:
```suggestion
_req_interceptor = cast(interceptor.RequestInterceptor, None)
```

then we can get rid of the `# type: ignore` comments because we essentially tell mypy "even though we set this to `None`, it's initialized to a value very early, so you can expect this to never actually be `None`"

Reply:
The problem is that `interceptor` is not imported at that point, it is imported later on in `webenginesettings.init`. mypy complains about that, and I coulnd't find a way of using something as a type before it is actually imported.

Reply:
You can pass it as a string instead, and add the import under a `if TYPE_CHECKING:` (via `from typing import TYPE_CHECKING`)

Reply:
Ah, perfect, thanks. Like this? b1bf969478e4b64d1f0871fd402ac5822d06286c